docs/en/stack/data-frames/dataframe-limitations.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[ml-df-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-limitations-wizard]]
+=== {dataframe-cap} wizard limitation
+
+The {dataframe} wizard in {kib} cannot display the value of a filed selected by 
+using the source index table on the right side if the selected field contains an 
+array.
+
+[float]
+[[df-limitations-ccs]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.2 for {dataframe-transforms}.
+
+[float]
+[[df-limitations-kibana]]
+=== Displaying {dataframe-transforms} in {kib} limitation
+
+{kib} displays {dataframe-transforms} up to 100.
```

Comment:
```suggestion
The {kib} *Machine Learning* > *Data Frames* page lists up to 100 {dataframe-transforms}.
```

Reply:
I think it's likely worth mentioning that you can work-around this limitation by calling the get data frame transforms API with the `size` parameter.